src/library/scala/runtime/Tuple2Zipped.scala

```diff
@@ -119,7 +119,7 @@ object Tuple2Zipped {
                 w2: T2 <:< CC2[El2],
                 bf: scala.collection.generic.CanBuildFrom[CC1[_], (El1, El2), That]
       ): That = {
-        val buf = bf(x._1)
+        val buf = bf(w1(x._1).asInstanceOf[CC1[_]])
```

Comment:
It looks like `CC1[El1]` is not a subtype `CC1[_]`. Is this an intended change? Or just a bug.